docs/document-qa.md

```diff
@@ -0,0 +1,116 @@
+# Document Question and Answering
+
+## Overview
+
+One of the most popular use cases for LLMs is doing Q/A over a large corpus of unstructured data.
+
+Fixie makes it possible to quickly build Q/A Agents by automatically crawling a set of developer-provided URLs, generating embeddings, chunking the data, and storing it inside of a Vector Database for efficient retrieval.
+
+To understand how to get started, let's look at a quick example of a [CodeShotAgent](/agents.md/#codeshotagent) that is designed to answer questions about Fixie:
+
+```python
+import fixieai
+
+BASE_PROMPT = """I am a helpful assistant that can answer questions \
+  about building on top of Fixie.ai. I try to be as concise as possible \
+  in my answers while still effectively answering the question."""
+
+URLS = [
+    "https://docs.fixie.ai/*"
+]
+
+DOCUMENTS = [fixieai.DocumentCorpus(urls=URLS)]
```

Comment:
What would it mean to have multiple members of this array?

Reply:
Good question for @juberti